pkg/startup/v11/data/master/etc/origin/master/master-config.yaml

```diff
@@ -1,5 +1,10 @@
 admissionConfig:
   pluginConfig:
+    ValidatingAdmissionWebhook:
```

Comment:
curious if you are sure that this is working? cf https://bugzilla.redhat.com/show_bug.cgi?id=1635918 ?

Reply:
Verified that the Validating admission webhook is enabled with this config and works. The creation of pods outside of the whitelist is blocked.